web-app/models/plant.js

```diff
@@ -0,0 +1,37 @@
+const { Sequelize } = require("sequelize");
+const sequelize = require("../app.js");
+const Moisture = require('./moisture.js');
+
+Plant = sequelize.define("plant", {
+  id: {
+    type: Sequelize.INTEGER,
+    autoIncrement: true,
+    allowNull: false,
+    primaryKey: true,
+  },
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  location: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  MAC: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  }
+});
+//Define 1-Many relationship
+Plant.hasMany(Moisture, {
```

Comment:
This is not the correct place for these relationships. Again, they need to be in a property called 'associate:'. Look for examples in the Sequelize docs. The code on lines 28-32 cannot handle these because they are not in the correct place.